projects/igniteui-angular/src/lib/dialog/dialog-content.component.html

```diff
@@ -6,8 +6,10 @@
         </div>
         <ng-content *ngIf="!title" select="igx-dialog-title,[igxDialogTitle]"></ng-content>
 
-        <div class="igx-dialog__window-content" *ngIf="message">{{ message }}</div>
-        <ng-content *ngIf="!message"></ng-content>
+        <div class="igx-dialog__window-content">
+            {{ message }}
```

Comment:
Isn't message deprecated for a while? If yes, then shouldn't it get removed?

Reply:
it is not deprecated.